lib/presentation/view/custom_list_tiles/shared_space_member_list_tile_builder.dart

```diff
@@ -29,66 +29,90 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 
+import 'package:domain/domain.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/src/material/list_tile.dart';
 import 'package:flutter/widgets.dart';
 import 'package:linshare_flutter_app/presentation/util/extensions/color_extension.dart';
 import 'package:linshare_flutter_app/presentation/view/avatar/label_avatar_builder.dart';
 
 typedef SelectRoleCallback = void Function();
+typedef DeleteMemberCallback = void Function();
 
 class SharedSpaceMemberListTileBuilder {
   final String _name;
   final String _email;
   final String _roleName;
+  final bool allowUpdate;
+  final bool allowDelete;
   final SelectRoleCallback onSelectedRoleCallback;
+  final DeleteMemberCallback onDeleteMemberCallback;
 
   Color tileColor;
 
-  SharedSpaceMemberListTileBuilder(this._name, this._email, this._roleName, {this.tileColor, this.onSelectedRoleCallback});
+  SharedSpaceMemberListTileBuilder(this._name, this._email, this._roleName,
+      {@required this.allowUpdate,
+      @required this.allowDelete,
```

Comment:
I don't like these boolean, how about when we have other function related to member? How about transfer the current role (optional) for this builder?